packages/browser/src/components/readers/imageBased/container/ReaderFooter.tsx

```diff
@@ -36,83 +39,104 @@ export default function ReaderFooter() {
 		[currentPage, pageSets],
 	)
 
+	const showToolBarChanged = usePreviousIsDifferent(showToolBar)
+	const readingDirectionChanged = usePreviousIsDifferent(readingDirection)
 	useEffect(() => {
 		if (showToolBar) {
 			virtuosoRef.current?.scrollToIndex({
 				align: 'center',
-				behavior: 'smooth',
+				behavior: showToolBarChanged || readingDirectionChanged ? 'auto' : 'smooth',
 				index: currentPageSetIdx,
 			})
 		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [showToolBar, currentPageSetIdx])
 
 	const formatDuration = useCallback(() => {
-		if (elapsedSeconds <= 60) {
-			return `${elapsedSeconds} seconds`
-		} else if (elapsedSeconds <= 3600) {
-			return dayjs.duration(elapsedSeconds, 'seconds').format('m [minutes] s [seconds]')
-		} else {
-			return dayjs
-				.duration(elapsedSeconds, 'seconds')
-				.format(`H [hour${elapsedSeconds >= 7200 ? 's' : ''}] m [minutes]`)
+		const duration = Duration.fromObject({ seconds: elapsedSeconds }).reconfigure({ locale })
+
+		let formattedDuration
+		if (elapsedSeconds <= 59) formattedDuration = duration.shiftTo('seconds')
+		else if (elapsedSeconds <= 3599) formattedDuration = duration.shiftTo('minutes', 'seconds')
+		else {
+			const hms = duration.shiftTo('hours', 'minutes', 'seconds')
+			formattedDuration = Duration.fromObject({ hours: hms.hours, minutes: hms.minutes })
 		}
```

Comment:
Can we use curly braces here? Whenever there is more than one conditional block and/or blocks that don't just return values I find it more readable and less error-prone